ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -24,24 +25,29 @@ ${MS_TABLE_STATUS_SUCCESS}=    //span[contains(@class,"pf-c-icon__content")][con
 ${MS_TABLE_STATUS_FAILURE}=    //span[contains(@class,"pf-c-icon__content")][contains(@class,"pf-m-danger")]
 ${KSERVE_MODAL_HEADER}=    //header[@class="pf-c-modal-box__header"]/h1[.="Deploy model"]
 ${KSERVE_RUNTIME_DROPDOWN}=    //span[.="Serving runtime"]/../../..//div[@id="serving-runtime-template-selection"]
+${LLM_RESOURCES_DIRPATH}=    ods_ci/tests/Resources/Files/llm
+${UWM_ENABLE_FILEPATH}=    ${LLM_RESOURCES_DIRPATH}/uwm_cm_enable.yaml
+${UWM_CONFIG_FILEPATH}=    ${LLM_RESOURCES_DIRPATH}/uwm_cm_conf.yaml
 
 
 *** Keywords ***
 Open Model Serving Home Page
     [Documentation]    Verifies submenu Settings > "Model Serving" is visible and navigates to page
-    Page Should Contain    Model Serving
-    Click Link      Model Serving
+    SeleniumLibrary.Page Should Contain    Model Serving
+    SeleniumLibrary.Click Link      Model Serving
     # Replace sleep with better Wait keyword. Confirm if error page has "Model Serving" h1 element.
     Sleep    1s
-    ${loading_error} =    Run Keyword And Return Status    Page Should Contain    Problem loading model serving page
+    ${loading_error}=    Run Keyword And Return Status
+    ...    SeleniumLibrary.Page Should Contain    Problem loading model serving page
     IF    ${loading_error}
         Log    "Page load error encountered"    level=WARN
-        Capture Page Screenshot
+        SeleniumLibrary.Capture Page Screenshot
         Sleep    5s
-        Reload Page
+        SeleniumLibrary.Reload Page
```

Comment:
Why using "SeleniumLibrary." in some of the keywords and imports ? (not only in this file)
Do we have some SeleniumLibrary keywords duplicated?

Reply:
there is duplication between SeleniumLibrary and JupyterLibrary. It works fine because we select the order of usage in the Suite setup, but last year it was discussed about it and my understanding is that the final decision was to explicitly call the library name to avoid IDEs to complain.